automates/apps/automates/model_code/sir_simple.py

```diff
@@ -0,0 +1,75 @@
+def drive(start, end, step, parameters):
+    step_results = {"S": list(), "I": list(), "R": list(), "dt": list()}
+
+    for i in range(start, end + 1, step):
+        (S, I, R) = sir(
+            parameters["S"],
+            parameters["I"],
+            parameters["R"],
+            parameters["beta"],
+            parameters["gamma"],
+            i,
+        )
+
+        step_results["S"].append(S)
+        step_results["I"].append(I)
+        step_results["R"].append(R)
+        step_results["dt"].append(i)
+
+    return step_results
```

Comment:
@dpdicken I think the keys in the final returned results should be GroMEt port IDs. Using the generated gromet for Simple_SIR I think we should hard-code these values to be:
```
"SIR-simple::SIR-simple::sir::0::--::s::0"
"SIR-simple::SIR-simple::sir::0::--::i::0"
"SIR-simple::SIR-simple::sir::0::--::r::0"
```